app/src/main/java/hr/dice/filipbionda/tmdbpractice/ui/showscreen/ShowScreen.kt

```diff
@@ -0,0 +1,671 @@
+){
+     val brush =
+        Brush.verticalGradient(
+            colorStops =
+            arrayOf(
+                0.0f to MaterialTheme.colorScheme.onBackground,
+                0.95f to MaterialTheme.colorScheme.surfaceContainer,
+            ),
+        )
+
+    Box(
+        modifier = modifier
+    ){
+        AsyncImage(
+            model = imagePath,
+            modifier = Modifier.fillMaxSize(),
+            contentDescription = null,
+            contentScale = ContentScale.Crop,
+            placeholder = painterResource(R.drawable.placeholder_image)
+        )
+        Box(
+            modifier = Modifier
+                .fillMaxSize()
+                .background(brush = brush)
+        )
+    }
+}
+
+@Composable
+private fun ShowScreenMediaBasicInfo(
+    title:String, date:String,
+    language:String, progress: Int,
+    duration: String?, seasons:Int?,
+    modifier: Modifier = Modifier
+) {
+
+    Row(
+        modifier = modifier,
+        verticalAlignment = Alignment.CenterVertically,
+        horizontalArrangement = Arrangement.Center
+    ) {
+        Box(
+            contentAlignment = Alignment.Center
+        ) {
+            CircularProgressIndicator(
+                progress = { progress/100f},
+                modifier = Modifier.size(72.dp),
+                trackColor = MaterialTheme.colorScheme.secondaryContainer,
+                color = MaterialTheme.colorScheme.secondary,
+                strokeWidth = dimensionResource(R.dimen.circular_progress_indicator_stroke_width)
+            )
+            Text(
+                text = "$progress%",
+                style = MaterialTheme.typography.bodyMedium
+            )
+        }
+        Spacer(
+            modifier = Modifier.width(dimensionResource(R.dimen.show_screen_media_info_spacer_width_1))
+        )
+        Column(
+            modifier = Modifier.fillMaxWidth()
+        ) {
+            val year = getYearFromDateString(date)
+            Text(
+                text = "$title ($year)",
+                style = MaterialTheme.typography.titleLarge.copy(
+                    fontSize = 24.sp,
+                ),
+                maxLines = 1
+            )
+            Spacer(
+                modifier = Modifier.height(dimensionResource(R.dimen.show_screen_media_info_spacer_height_1))
+            )
+            Row(
+                modifier = Modifier.fillMaxWidth(),
+                verticalAlignment = Alignment.CenterVertically
+            ) {
+                Text(
+                    text = "$date ($language)",
+                    style = MaterialTheme.typography.bodySmall.copy(
+                        fontSize = 16.sp,
+                        color = MaterialTheme.colorScheme.onSurface
+                    )
+                )
+                Spacer(
+                    modifier = Modifier.width(dimensionResource(R.dimen.show_screen_media_info_spacer_width_2))
+                )
+                if (duration!=null){
```

Comment:
make duration non null for now we can deal with that layer and then you can remove this if statement

Reply:
Why? Logic behind this is if there is duration that means movie is displayed and if there are seasons we have serie, soap or etc. If this is bad logic I will remove it.